docs/site/Model.md

```diff
@@ -230,26 +216,48 @@ now:
   </thead>
 
   <tbody>
-
   <tr>
-  <td><code>description</code></td>
+    <td><code>name</code></td>
+    <td>String</td>
+    <td>None</td>
```

Comment:
nitpick: shall we keep the default value consistent with other fields as `N/A` instead of `None`?